frontend/src/app/home/map/map.component.ts

```diff
@@ -43,21 +46,22 @@ export class MapComponent implements OnInit {
 
   coordinates = dummyCoordinates;
 
+  private langChangeSubscription: Subscription;
+
   constructor(
     private readonly geolocationService: GeolocationService,
     private readonly snackService: SnackService,
     private shopService: ShopService,
     private translateService: TranslateService,
-    private logger: NGXLogger
-  ) {
-    this.options = this.geolocationService.getMapOptions();
-    this.geolocationService.geolocationTriggerEvent$.subscribe(() => {
-      this.locateMe();
-    });
+    private logger: NGXLogger) {
+      this.options = this.geolocationService.getMapOptions();
+      this.geolocationService.geolocationTriggerEvent$.subscribe(() => {
```

Comment:
this subscription should be also cleaned up

Reply:
This has been garbage collected; good catch!